src/simularium/RemoteSimulator.ts

```diff
@@ -314,7 +318,7 @@ export class RemoteSimulator implements ISimulator {
         this.logger.debug("Web Socket Request Sent: ", whatRequest, jsonData);
     }
 
-    private sendWebSocketRequest(jsonData, requestDescription): void {
+    public sendWebSocketRequest(jsonData, requestDescription: string): void {
```

Comment:
Needed this public for testing

Reply:
I wonder if there's a way to let a thing stay private but make it public for unit tests

Reply:
I couldn't find a way to do that, but I modified the test so that we don't have to access this method.

Reply:
Cool.  You can do it in C++ by using the "friend" keyword which allows other classes access to my class's private members.

Reply:
Whaaaaat that's pretty cool.